AardvarkLoggingUI.podspec

```diff
@@ -0,0 +1,17 @@
+Pod::Spec.new do |s|
+  s.name     = 'AardvarkLoggingUI'
```

Comment:
It'd be good to land an update to the README mentioning `AardvarkLoggingUI` as part of this PR.

Reply:
Planning on following up with updating the README and other documentation to reflect the new module structure.